imitation/util/util.py

```diff
@@ -89,14 +88,14 @@ def __init__(self, *args, **kwargs):
                      net_arch=[32, 32], feature_extraction="mlp")
 
 
-@gin.configurable
 class FeedForward64Policy(FeedForwardPolicy):
   def __init__(self, *args, **kwargs):
     super().__init__(*args, **kwargs,
                      net_arch=[64, 64], feature_extraction="mlp")
 
 
-@gin.configurable
+@train_exp.capture(prefix="make_blank_policy_kwargs")
```

Comment:
Consider making `make_blank_policy` an ingredient, which both experiments depend on. https://sacred.readthedocs.io/en/latest/ingredients.html

You could alternately just not make it depend on Sacred at all, and feed in the arguments manually from `train` and `data_collect`. The `capture` style can avoid huge parameter lists making things more readable, but it also makes it hard to understand where all your parameters have an effect. Because of this I like to try and limit the scope of Sacred `capture` and `config` to my script/experiment files only.

Reply:
I limited `capture` to my script file